### ISSUE TITLE
Proper Dutch language (nl-nl) support

### ISSUE THREAD

**Issue by mikewoudenberg** [MycroftAI/mycroft-core#2380]
At this moment Mycroft already contains a lot of translations for the Dutch language. It already contains logic for formatting Dutch date, times and numbers. It is however lacking support for parsing Dutch sentences, which causes a number of built in skills to fail because of the unsupported language. I'm working on extending this parse functionality and this issue is to help track efforts in doing so.

**Comment by forslund** [comment]
Thank you for picking this up and improving the internationalization. Before we can accept your work we need you to sign a CLA. There is some manual intervention needed on our side so you might want to start the processes now. Go to https://mycroft.ai/cla to get going!

**Comment by krisgesling** [comment]
Thanks for your efforts Mike!
If you have any questions, or there are other ways we can help, please let us know :)

**Comment by j1nx** [comment]
Laat me weten als en waarmee je hulp kunt gebruiken.

(Let me know if and with which you can use some help.)

**Comment by forslund** [comment]
With #2385 merged could you update the issue with specific parts that needs further work?

**Comment by j1nx** [comment]
With the following section in mycroft.conf

```
{
  "max_allowed_core_version": 19.8,
  "tts":{"google":{"lang": "nl"}},
  "stt":{"module":"google", "google":{"lang": "nl"}},
  "lang":"nl-nl"
}
```
I can ask for the weather in Dutch and get a response back in Dutch. Don't know which other skills have already been translated, but I would consider; "Addition of rudimentary Dutch support" as done and this can be closed.

Now it is just a matter of translating skills and other sections....

**Comment by mikewoudenberg** [comment]
Yup, with my PR merged and #2411 merged it is working quite nicely right now thanks!
